sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md

```diff
@@ -42,210 +38,208 @@ In the case of Service Bus, the modern client libraries have packages and namesp
 
 ### Client hierarchy
 
-In the interest of simplifying the API surface we've made a single top level client called `ServiceBusClient`, rather than one for each of queue, topic and subscription:
+In the interest of simplifying the API surface we've made a single top level client called `ServiceBusClient`, rather than one for each of queue, topic, subscription and session. This acts as the single entry point in contrast with six different entry points in the previous library. You can create senders and receivers off of this client to the queue/topic/subscription/session of your choice and start sending/receiving messages.
+
+#### Approachability
+By having a single entry point, the `ServiceBusClient` helps with the discoverability of the API as you can dot into the client and see all the available methods as opposed to searching through documentation or exploring namespace for the types that you can instantiate. Whether sending or receiving, or using sessions or not, users will start their applications by constructing the same client.
```

Comment:
nit:  This doesn't read correctly to me. - specifically, I don't think we want to use  `dot` as a verb.  We probably don't want to exclude properties from that statement.  Perhaps something like:

`... you can use your IDE to explore the members...`

Reply:
fair point, I was shamelessly piggy backing on @JoshLove-msft's work from another issue :)
Updated in https://github.com/Azure/azure-sdk-for-net/pull/14166/commits/a02534f2b496a08d65a71effcf9e7d1de7136f1c